images/macos/helpers/Xcode.Installer.psm1

```diff
@@ -86,7 +83,7 @@ function Expand-XcodeXipArchive {
         [string]$TargetPath
     )
 
-    $xcodeXipPath = Get-ChildItem -Path $DownloadDirectory -Filter "Xcode_*.xip" | Select-Object -First 1
+    $xcodeXipPath = Get-ChildItem -Path $DownloadDirectory -Filter "Xcode-*.xip" | Select-Object -First 1
```

Comment:
nit: may be keep old naming with underscore?
I think it is a bit clearer `Xcode_13.1-Beta2+13A1030d`

Reply:
those name generated by `xcodes`, I'd leave its naming scheme